src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/CodeStyle/NotificationOption2.cs

```diff
@@ -20,64 +20,48 @@ namespace Microsoft.CodeAnalysis.CodeStyle
     /// <completionlist cref="NotificationOption2"/>
     internal sealed partial class NotificationOption2 : IEquatable<NotificationOption2?>
```

Comment:
This class now stores the severity only. Can it be deleted entirely and let all consumers deal with `ReportDiagnostic` value directly?

Reply:
Will follow up.